playbooks/openstack/inventory.py

```diff
@@ -124,11 +124,10 @@ def build_inventory():
     '''Build the dynamic inventory.'''
     cloud = shade.openstack_cloud()
 
-    # TODO(shadower): filter the servers based on the `OPENSHIFT_CLUSTER`
-    # environment variable.
+    cluster_name = os.getenv('OPENSHIFT_CLUSTER', 'openshift-cluster')
```

Comment:
Would it be possible to isolate this in a function, replacing the two instances where we use this code?